doc/getting-started/how-helix-works.md

```diff
@@ -0,0 +1,19 @@
+# How does Helix work?
+
+This page takes a behind-the-scenes look at the overall Helix architecture and describes its main components.
+
+![Helix Architecture Overview](/assets/helix-overview.png)
```

Comment:
Some thoughts on the diagram itself:

- Unless our target helix developer demographic is mostly for people who use right-to-left written languages, I would flip the diagram horizontally, and put the client on the left and GitHub on the right.
- I would put the diagram 'column' labels on the top, not on the bottom. The labels like "Fastly CDN", "GitHub", "Adobe I/O Runtime"
- I would label the column containing the client (perhaps "Client"?).
- The helix commands column could maybe be integrated into the Adobe I/O Runtime column somehow?

Reply:
I have completely redone the diagram and it's top to bottom now, with light grey boxes to group things.